internal/environment/command_update.go

```diff
@@ -19,10 +19,12 @@ func (c *command) newUpdateCommand() *cobra.Command {
 	}
 
 	cmd.Flags().String("name", "", "New name for Confluent Cloud environment.")
+	cmd.Flags().String("stream-governance-package", "",
+		`Stream Governance package. "essentials" or "advanced"`)
```

Comment:
Let's use `utils.ArrayToCommaDelimitedString()` in the description, and add autocompletion for this flag (make a helper function called `c.addStreamGovernancePackageFlag()` since this flag is used in more than one command)